UITable/UITable/DataStructureViewController.swift

```diff
@@ -27,17 +31,28 @@ class TableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return arrayDataStructure.count
+        return modelDataStructure.count()
     }
 
-    
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as! TableViewCell
-        let curentTextInCell = arrayDataStructure[indexPath.row]
+        
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath) as? DataStructureViewCell else {
+            fatalError("The dequeued cell is not an instance of DataStructureViewCell.")
```

Comment:
Wow, don't kill school students who came without copybook! Don't kill application just because one have forgot to add cell identifier.